corporate-connections/app/models/CompaniesHouseModels.scala

```diff
@@ -1,7 +1,7 @@
 package models
 
-case class CompanyLink(company: String)
-case class Appointment(officer_role: String, links: CompanyLink)
+case class AppointedTo(company_name: String, company_number: String, company_status: String)
+case class Appointment(officer_role: String, appointed_to: AppointedTo)
```

Comment:
this is much more comprehensible now that we're using the more descriptive bit of the API